lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,6 +2,11 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        def initialize(serializer, options = {})
+          super
+          @include_tree = IncludeTree.from_include_args(options[:include] || '*')
```

Comment:
should just options[:include] be passed, and then in the method, do `include || '*'`